apps/user-service/src/main/resources/application-dev.yaml

```
@@ -0,0 +1,3 @@
+spring:
+  jpa:
+    show-sql: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)